katago_server/games/managers/rating_game_pandas_manager.py

```diff
@@ -1,15 +1,23 @@
+import logging
 import pandas
 from django.db.models import Manager
 from django_pandas.io import read_frame
 
 from katago_server.games.managers.rating_game_pandas_queryset import RatingGamePandasQuerySet
 from katago_server.runs.models import Run
 
+logger = logging.getLogger(__name__)
```

Comment:
Nice, I will make a pass to improve logging. In production we are blind right now.

I'm planning to use https://www.structlog.org/en/stable/getting-started.html
so in production we have strured log rather than text (it easier to index a json thata text).